### DISCUSSION TITLE
Repo is getting archived

### DISCUSSION THREAD

**Discussion by cromewar** [smartcontractkit/full-blockchain-solidity-course-py#1872] [Announcements]
We hope this announcement finds you well. We wanted to take a moment to let you know that the repository related to the "Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial" will be archived soon. As much time has passed since the course was published, we have decided that it's time to archive the repository.

We would like to take this opportunity to thank all of you who participated in the course by asking questions and engaging in discussions. Your contributions were invaluable to the success of the course, and we appreciate your enthusiasm and dedication.

In particular, we would like to extend a special thank you to @PatrickAlphaC for creating the course. Your hard work and dedication to educating the community on Solidity, blockchain, and smart contract development have not gone unnoticed.

For those of you who are interested in continuing to learn more about these topics, we encourage you to check out the JavaScript version of the course, which is still supported and available for you to explore.

Once again, thank you to everyone who contributed to the success of this course. We look forward to continuing to work with you all in the future.

Best regards,
Vasiliy

**Comment by PatrickAlphaC** [comment]
thanks for this @cromewar 

For those looking for next steps, be sure to subscribe here: https://web3education.dev/

**Comment by Alain-Pascal** [comment]
please share link to the javascript version
